js_modules/dagster-ui/packages/ui-core/src/graphql/schema.graphql

```diff
@@ -2879,7 +2879,6 @@ type PartitionBackfill implements RunsFeedEntry {
   partitionNames: [String!]
   isValidSerialization: Boolean!
   numPartitions: Int
-  numCancelable: Int!
```

Comment:
Did you get an answer about the push safety of this? I would say that it's not push-safe as-is, and that you should remove the field from existing queries and get that into prod for a couple days before you try to remove the field from the backend.

Reply:
Oh hmm that's interesting, yeah if the deploy is not immediate on both sides it would fail for a few min. I will go ahead and split it out (probably should have done that initially so this could merge 2 weeks ago 🫠 oof)